hole/__init__.py

```diff
@@ -34,7 +35,7 @@ def __init__(
         self.host = host
         self.location = location
         self.api_token = api_token
-        self.data = {}
+        self.data: dict[str, Any] = {}
```

Comment:
This requires `from __future__ import annotations`.
```suggestion
        self.data: dict[str, Any] = {}
```

Reply:
correct - however HA uses 3.7 at its minimum so I don't think thats a problem